effekt/shared/src/main/scala/effekt/Compiler.scala

```diff
@@ -66,6 +66,17 @@ export PhaseResult.*
 
 enum Stage { case Core; case Lifted; case Machine; case Target; }
 
+/**
+ * Constants for common feature flags (not specific to one generator)
+ */
+object FeatureFlag {
+  /** For backends that translate to a programming language supporting at least:
+   * - directly splicing in variables ("${...}" works)
+   * - "simple" literals like "42"
+   */
+  val simpleAtom: String = "simpleAtom"
```

Comment:
Here, I now define common feature flags (currently one).
`simpleAtom` is meant to be supported by backends that translate such that both simple literals (ints like "42") and variable names (mostly for "${x}") are allowed.

Reply:
(Removed)